src/DataStructures/DataBox/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+spectre_target_headers(
+  DataStructures
+  INCLUDE_DIRECTORY ${CMAKE_SOURCE_DIR}/src
+  HEADERS
+  DataBoxHelpers.hpp
+  DataBox.hpp
+  DataBoxTag.hpp
```

Comment:
In these `CMakeLists.txt` files, you aren't consistent about the order of `.` vs letters. I suggest matching the C++ and putting the `.` first. (So `DataBox.hpp` above `DataBoxHelpers.hpp`)